_data/toc.yaml

```diff
@@ -1389,14 +1387,18 @@ manuals:
   section:
     - path: /build/
       title: Overview
-    - path: /build/hellobuild/
-      title: Hello Build
+    - sectiontitle: Building images
+      section:
+        - path: /build/building/packaging/
+          title: Packaging your software
+        - path: /build/building/multi-platform/
+          title: Multi-platform images
     - sectiontitle: Buildx
       section:
         - path: /build/buildx/
-          title: Buildx overview
+          title: Overview
```

Comment:
I'd like to get rid of the Buildx section if we can, but maybe we could approach that later.

- We should merge the multiple builders page into the drivers overview, since it's too small by itself, and duplicates some of the info.
- Buildx verview page could be merged to the build overview
- Buildx install could be made to a generic build install page, where we could also cover buildkit as well later

Then drivers is left by itself, so I think we should move it to Building Images.

Reply:
Yes we should get ride of it and move bits around.

> We should merge the multiple builders page into the drivers overview, since it's too small by itself, and duplicates some of the info.

Yes that sounds good :+1:

> Buildx verview page could be merged to the build overview

As discussed yes good point

> Buildx install could be made to a generic build install page, where we could also cover buildkit as well later

Yes some sort of `Install` section with Buildx, BuildKit and other tooling looks good.

> Then drivers is left by itself, so I think we should move it to Building Images.

:+1:

Reply:
Merged buildx overview to main overview page. For multiple builders, I think you can move some bits to driver guide in buildx repo.